httpserver/http_connection.cpp

```diff
@@ -480,6 +483,34 @@ void connection_t::process_retrieve_all() {
     response_.result(http::status::ok);
 }
 
+bool connection_t::check_correct_swarm(std::string pub_key) {
```

Comment:
I would rename this to `handle_incorrect_swarm` or `handle_wrong_swarm` and invert the return value, to make it return true if it did handle it.
```cpp
if (handle_wrong_swarm(pubkey)) {
  return;
}
```

Reply:
Actually I like Maxim's suggestion better:
```cpp
if (!service_node_.is_pubkey_for_us(pub_key)) {
  handle_wrong_swarm(pub_key);
  return;
}
```